website/docs/configuration.md

```diff
@@ -27,6 +27,17 @@ let g:neovide_refresh_rate=140
 Setting `g:neovide_refresh_rate` to a positive integer will set the refresh rate of the app. This is
 limited by the refresh rate of your physical hardware, but can be lowered to increase battery life.
 
+#### Idle Refresh Rate
+
+```vim
+let g:neovide_refresh_rate_idle=10
```

Comment:
The configuration docs document the default values, but here it's listed at `10` while the `Default` implementation of `WindowSettings` lists `5`.

Reply:
Thank you for having a look!
I thought a non default value was a deliberate choice there since the default value for `refresh_rate` is 60 and not 140 as in the lines above.
I'll fix both.

Reply:
Oh, sorry from our side then!

Reply:
No reason to apologize for!